managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1503,7 +1503,7 @@ public void operationComplete(Void v, Stat stat) {
                     currentLedgerEntries = 0;
                     currentLedgerSize = 0;
                     metadataMutex.unlock();
-                    updateLedgersIdsComplete(stat);
+                    updateLedgersIdsComplete();
```

Comment:
This is not a fix for a flaky test, 
this is a change on ManagedLedgerImpl

I think that we should change the title

Reply:
after re-reading the code I have verified that the behaviour did not change.
you only removed a useless parameter and split the method

Reply:
so there is no need to change the title